azurerm/resource_arm_key_vault_test.go

```diff
@@ -648,3 +670,78 @@ resource "azurerm_key_vault" "test" {
 }
 `, rInt, location, rInt)
 }
+
+func testAccAzureRMKeyVault_accessPolicyUpperLimit(rInt int, location string) string {
+	
+	var storageAccountConfigs string
+	var accessPoliciesConfigs string
+	var dependsString string
+        var maxPolicies = 68
```

Comment:
could we fix the indentation, it could also be rewritten as `maxpolicies := 68` and could we change this to a `numberPolicies`?

Reply:
Removed maxPolicies